docs/source/developer/contributing.rst

```diff
@@ -252,6 +252,25 @@ A code review should try to assess the following criteria:
 * docstrings, comments
 * test coverage
 
+
+Policy on large scale changes to code formatting
+''''''''''''''''''''''''''''''''''''''''''''''''
+
+Please note that pull requests making large scale changes to format the code
+base are in general not accepted. Such changes often increase the likelihood of
+merge conflicts for other pull requests, which inevitably take time and
+resources to resolve. They also require a lot of effort to check as Numba aims
+to compile code that is valid even if it is not ideal, i.e. testing
+``operator.eq``::
+
+    if x == None: # should be `if x is None:` but it's valid so don't change it!
+
+This policy has been adopted by the core developers so as to try and make best
+use of limited resources. Whilst it would be great to have an extremely tidy
+code base, priority is given to fixes and features over code formatting changes.
+Thanks for your understanding on this matter!
```

Comment:
I think this comes from a place of genuine gratitude, but I'm concerned it reads a bit passive-aggressively. I also think the explanation up to this point was a clear and calm explanation of the situation, so perhaps it is unnecessary?

```suggestion
```

Reply:
Thanks for raising this, in the interests of ensuring that it cannot be interpreted with a meaning/in a manner that is unintended, I'll remove it.